src/Installers/Windows/AspNetCoreModule-Setup/ANCMIISExpressV2/ancm_iis_expressv2.wxs

```diff
@@ -164,22 +164,63 @@
       <DirectoryRef Id="INSTALLDIR">
         <Directory Id="INSTALLLOCATION" ShortName="ANCM" Name="$(var.ProductName)">
           <Directory Id="VersionDir" Name="$(var.ProductVersionString)">
-            <?if $(var.Platform) = "arm64" ?>
-            <Component Id="AspNetCoreModuleV2.forwarder"
-                Guid="08968573-05c1-4bf1-8879-7b818ac9525b"
-                Win64="$(var.IsWin64)">
-                <File Id="AspNetCoreModuleV2Dll.forwarder"
+            <Component Id="AspNetCoreModule" Guid="84ed6ce6-c8a3-4fa8-a872-c98a1d15dd4f" Win64="$(var.IsWin64)">
+                <?if $(var.Platform) = "arm64" ?>
+                <File Id="AspNetCoreModuleDll"
                     Name="aspnetcorev2.dll"
                     Source="$(var.ArtifactsDir)\bin\AspNetCoreModuleForwarders\aspnetcorev2.dll"
                     DiskId="1"
                     Vital="yes"/>
-                <RegistryKey Root="HKLM"
-                    Key="SYSTEM\CurrentControlSet\Services\EventLog\Application\$(var.ProductShortName)">
-                    <RegistryValue Name="EventMessageFile" Type="expandable"
-                        Value="[#AspNetCoreModuleV2Dll.forwarder]" />
-                    <RegistryValue Name="TypesSupported" Type="integer" Value="7" />
-                </RegistryKey>
+                <?else ?>
+                <File Id="AspNetCoreModuleDll"
+                    Name="aspnetcorev2.dll"
+                    Source="$(var.AspNetCoreV2ProgramFilesTargetPath)"
+                    DiskId="1"
```

Comment:
`File@DiskId="1"` and `File@Vital="yes"` are the defaults, there are probably more of these to clean up